pkg/cmd/logs.go

```diff
@@ -0,0 +1,235 @@
+// Copyright KubeArchive Authors
+// SPDX-License-Identifier: Apache-2.0
+package cmd
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/client-go/rest"
+)
+
+type LogsOptions struct {
+	ContainerName string
+	Name          string
+	Resource      string
+	GroupVersion  string
+	LabelSelector string
+
+	Token string
+
+	KubeArchiveHost string
+	KubeArchiveCA   string
+
+	RESTConfig *rest.Config
+	kubeFlags  *genericclioptions.ConfigFlags
+}
+
+var logsLong = `Print the logs for a container in a pod or specified resource from KubeArchive.
+If the provided resource owns directly or indirectly a pod, and all the resources in the chain are archived
+you can also get the logs one of the containers in the chain (latest one if it is not specified).`
```

Comment:
I don't understand 100% this paragraph. what do you think about this:
`Print the logs for a container in a pod or specified resource from KubeArchive.
If the provided resource, directly or indirectly, owns a pod, and all the resources in the chain are archived,
you can also retrieve the logs of one of the containers in the chain (the latest one, if not specified).`